vr-je-palette-picker/palette-picker-back-end
# Seeds and endpoints

## What does this PR do?
This PR migrates and seeds the data on both the development and testing environments, alongside installing necessary testing packages.

### Where should the reviewer start?
In the db directory, please review both dev and test directories for the seeding files. After that if you could look over app.js and server.js to see if there are any mistakes I have made and not caught! Thank you!

#### How should this be manually tested?
When you approve and pull down, please check your Postico to see if the tables in both databases have the data we seeded together tonight.

#### Any background context you want to provide?


#### What are the relevant tickets?
Closes #34 
Closes #35 
Closes #36 
Closes #37 

#### Screenshots (if appropriate)
![Seeded data, will re-migrate and re-seed before deployment to ensure ids start at 1](https://user-images.githubusercontent.com/31895658/70109371-e83bbd00-1608-11ea-9760-ac14691fc8db.png)
<img width="407" alt="Screen Shot 2019-12-03 at 8 10 59 PM" src="https://user-images.githubusercontent.com/31895658/70109415-0f928a00-1609-11ea-9684-d03eb3286537.png">

#### Additional Notes
Thank you for your hard work today! I really appreciate your dedication.
